cmd/zoekt-sourcegraph-indexserver/main_test.go

```diff
@@ -112,6 +114,7 @@ func TestListRepoIDs_Error(t *testing.T) {
 func TestMain(m *testing.M) {
 	flag.Parse()
 	if !testing.Verbose() {
+		logtest.InitWithLevel(m, zoektlog.LevelNone)
```

Comment:
I tested `logtest.InitWithLevel(m, zoektlog.LevelDebug)` for Verbose test runs and the log output is present, so we have the option to add this log init to `TestMain()`. However, for now I left it out for now to make test output conservative.

Here's the test output when running verbose flag:

```
=== RUN   TestIndex

=== RUN   TestIndex/minimal
INFO zoekt-sourcegraph-indexserver/index.go:278 successfully fetched git data {"repo": "test/repo", "id": 0, "commits_count": 1, "duration": "1.339µs"}
=== RUN   TestIndex/minimal-id
INFO zoekt-sourcegraph-indexserver/index.go:278 successfully fetched git data {"repo": "test/repo", "id": 123, "commits_count": 1, "duration": "907ns"}

=== RUN   TestIndex/all
INFO zoekt-sourcegraph-indexserver/index.go:278 successfully fetched git data {"repo": "test/repo", "id": 0, "commits_count": 2, "duration": "10.637µs"}

=== RUN   TestIndex/delta
INFO zoekt-sourcegraph-indexserver/index.go:278 successfully fetched git data {"repo": "test/repo", "id": 0, "commits_count": 6, "duration": "1.479µs"}

--- PASS: TestIndex (0.00s)
```

Note that our Development runs do not include the `otfields.Resource`